YOLayout/YOCGUtils.m

```diff
@@ -64,9 +64,9 @@ CGRect YOCGRectBottomAlignWithRect(CGRect rect, CGRect inRect) {
   return CGRectMake(rect.origin.x, y, rect.size.width, rect.size.height);
 }
 
-CGRect YOCGRectToCenterInRect(CGSize size, CGRect inRect) {
-  CGPoint p = YOCGPointToCenter(size, inRect.size);
-  return CGRectMake(p.x + inRect.origin.x, p.y + inRect.origin.y, size.width, size.height);
+CGRect YOCGRectToCenterInRect(CGRect rect, CGRect inRect) {
```

Comment:
I think its ok... I mean in this case origin is 0 but in some other scenario in the future it might not?